app/views/application_records/show.haml

```diff
@@ -6,6 +6,10 @@ by:
 = @record.user.name_and_email
 = button_to 'Print this page',
   application_record_path(@record, format: :pdf), method: :get
+- if !@record.saved_for_later && @record.note_for_later.present?
```

Comment:
Someday? Sure, but I don't write apps for someday. I make them for now, but future-proofed. 
It would be easy enough to add that attribute, but as it stands currently, there's always a note. If there's no note, someone would just have an apparently new application to read through. Not a big deal--they'll either come to the same conclusion they came to before, or they'll reject it based on new information.